src/surface_segmentation_action.cpp

```diff
@@ -175,6 +175,23 @@ void SurfaceSegmentationAction::Execute(
   for (size_t i = 0; i < surface_objects.size(); ++i) {
     const SurfaceObjects& surface_scene = surface_objects[i];
     num_objects += surface_scene.objects.size();
+
+    shape_msgs::SolidPrimitive surface_shape;
```

Comment:
As I mentioned in the rapid_pbd_msgs PR, I'd like to return the result in a custom message of type `rapid_pbd_msgs/Surface` instead of a `moveit_msgs/CollisionObject`.

The reason is that the only purpose of this action is to provide perceptual information, so the API should return a result that does not suggest a particular use of the information (like a CollisionObject).